controllers/hibp.js

```diff
@@ -81,12 +81,13 @@ async function notify (req, res) {
   log.info("notification", { length: recipients.length, breachAlertName: breachAlert.Name });
 
   const utmID = "breach-alert";
-  const ctaHref = EmailUtils.getViewMyDashboardHref(utmID);
   const notifiedRecipients = [];
 
   for (const recipient of recipients) {
     log.info("notify", {recipient});
     const { recipientEmail, breachedEmail, signupLanguage, preFxaSubscriber } = getAddressesAndLanguageForEmail(recipient);
+    const campaignId = (preFxaSubscriber) ? "preFXA-see-all-breaches" : "see-all-breaches";
+    const ctaHref = EmailUtils.getEmailCtaHref(utmID, campaignId, preFxaSubscriber);
```

Comment:
This call to `getEmailCtaHref` passes 3 arguments, but the function definition only has 2 parameters?